web/public/themes/nex.css

```
@@ -57,72 +57,74 @@ html[data-theme="nex"] {
+  --yellow-dark: var(--warning-500);
+  --yellow-bg: var(--warning-100);
 
-  --blue:           var(--cyan-500);
-  --blue-bg:        var(--cyan-100);
+  --blue: var(--cyan-500);
+  --blue-bg: var(--cyan-100);
 
   /* ── Typography ── */
-  --font-sans:  -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif;
+  --font-sans:
+    -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif;
```

Stylelint (17.10.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)